docs/notebooks/lda_training_tips.ipynb

```diff
@@ -67,7 +68,7 @@
     "    files = os.listdir(data_dir + yr_dir)\n",
     "    for filen in files:\n",
     "        # Note: ignoring characters that cause encoding errors.\n",
-    "        with open(data_dir + yr_dir + '/' + filen, errors='ignore') as fid:\n",
+    "        with smart_open(data_dir + yr_dir + '/' + filen, 'rb') as fid:\n",
```

Comment:
Bug (not in this PR, notebook already bad): use `os.path.join` for joining filesystem paths.